src/index.ts

```diff
@@ -27,6 +28,21 @@ interface Logger {
   trace (...msg: any[]): void
 }
 
+export interface IlpPluginMiniAccountsConstructorOptions {
+  port?: number
+  wsOpts?: WebSocket.ServerOptions
+  currencyScale?: number
+  debugHostIldcpInfo?: ILDCP.IldcpResponse
+  allowedOrigins?: string[]
+  generateAccount?: boolean
+  _store?: Store
+}
+
+export interface IlpPluginMiniAccountsConstructorModules {
+  log?: Logger
+  store?: StoreWrapper
```

Comment:
@tangsauce Changing this from `Store` to `StoreWrapper` was a breaking change, and breaks the build for plugins that extend it. Could you please revert/fix this?

Reply:
@kincaidoneil, this [PR](https://github.com/interledgerjs/ilp-plugin-mini-accounts/pull/47) should fix the bug. Thanks for bringing this up and I'll do my best to make sure it doesn't happen again.